orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Execution.groovy

```diff
@@ -45,6 +45,7 @@ abstract class Execution<T> implements Serializable {
   ExecutionStatus status = NOT_STARTED
 
   AuthenticationDetails authentication
+  PausedDetails paused
```

Comment:
@robfletcher I opted to nest the paused details rather than flatten them directly on the `Execution`. Thoughts? This is in contrast to what we have for canceled (_although it might make sense to change_).